pkg/metrics/otel.go

```diff
@@ -54,13 +54,18 @@ exporters:
   googlecloud/kubernetes:
     metric:
       prefix: "kubernetes.io/internal/addons/config_sync/"
-      skip_create_descriptor: false
+      skip_create_descriptor: true
+      instrumentation_library_labels: false
```

Comment:
Why do we need to set this field to false?
I don't think it increases cardinality.

Reply:
Example error `Unable to send metric descriptor.       {"kind": "exporter", "data_type": "metrics", "name": "googlecloud", "error": "rpc error: code = InvalidArgument desc = Request was missing field metricDescriptor.valueType: The descriptor does not have the value type set.", "metric_descriptor": "name:\"internal_errors_total\"  type:\"custom.googleapis.com/opencensus/config_sync/internal_errors_total\"  labels:{key:\"instrumentation_source\"}  labels:{key:\"instrumentation_version\"}  metric_kind:CUMULATIVE  unit:\"1\"  description:\"The total number of internal errors triggered by Config Sync\"  display_name:\"opencensus/config_sync/internal_errors_total\""}`. By default the exporter will try to attach these two labels: `instrumentation_source` and `instrumentation_version`, using this flag to disable the labels.

Reply:
Add a comment to explain this in the file.

Reply:
Done!